docs/api/ignite_v1alpha3.md

```diff
@@ -0,0 +1,492 @@
+# v1alpha3
+
+`import "github.com/weaveworks/ignite/pkg/apis/ignite/v1alpha3"`
+
+  - [Overview](#pkg-overview)
+  - [Index](#pkg-index)
+
+## <a name="pkg-overview">Overview</a>
+
+\+k8s:deepcopy-gen=package +k8s:defaulter-gen=TypeMeta
++k8s:openapi-gen=true
++k8s:conversion-gen=github.com/weaveworks/ignite/pkg/apis/ignite
+
+## <a name="pkg-index">Index</a>
+
+  - [Constants](#pkg-constants)
+  - [Variables](#pkg-variables)
+  - [func SetDefaults\_PoolSpec(obj \*PoolSpec)](#SetDefaults_PoolSpec)
+  - [func SetDefaults\_VMKernelSpec(obj
+    \*VMKernelSpec)](#SetDefaults_VMKernelSpec)
+  - [func SetDefaults\_VMSandboxSpec(obj
+    \*VMSandboxSpec)](#SetDefaults_VMSandboxSpec)
+  - [func SetDefaults\_VMSpec(obj \*VMSpec)](#SetDefaults_VMSpec)
+  - [type BlockDeviceVolume](#BlockDeviceVolume)
+  - [type FileMapping](#FileMapping)
+  - [type Image](#Image)
+  - [type ImageSpec](#ImageSpec)
+  - [type ImageStatus](#ImageStatus)
+  - [type Kernel](#Kernel)
+  - [type KernelSpec](#KernelSpec)
+  - [type KernelStatus](#KernelStatus)
+  - [type OCIImageSource](#OCIImageSource)
+  - [type Pool](#Pool)
+  - [type PoolDevice](#PoolDevice)
+  - [type PoolDeviceType](#PoolDeviceType)
+  - [type PoolSpec](#PoolSpec)
+  - [type PoolStatus](#PoolStatus)
+  - [type Runtime](#Runtime)
+  - [type SSH](#SSH)
+      - [func (s \*SSH) MarshalJSON() (\[\]byte,
+        error)](#SSH.MarshalJSON)
+      - [func (s \*SSH) UnmarshalJSON(b \[\]byte)
+        error](#SSH.UnmarshalJSON)
+  - [type VM](#VM)
+  - [type VMImageSpec](#VMImageSpec)
+  - [type VMKernelSpec](#VMKernelSpec)
+  - [type VMNetworkSpec](#VMNetworkSpec)
+  - [type VMSandboxSpec](#VMSandboxSpec)
+  - [type VMSpec](#VMSpec)
+  - [type VMStatus](#VMStatus)
+  - [type VMStorageSpec](#VMStorageSpec)
+  - [type Volume](#Volume)
+  - [type VolumeMount](#VolumeMount)
+
+#### <a name="pkg-files">Package files</a>
+
+[defaults.go](https://github.com/weaveworks/ignite/tree/master/pkg/apis/ignite/v1alpha3/defaults.go)
+[doc.go](https://github.com/weaveworks/ignite/tree/master/pkg/apis/ignite/v1alpha3/doc.go)
+[json.go](https://github.com/weaveworks/ignite/tree/master/pkg/apis/ignite/v1alpha3/json.go)
+[register.go](https://github.com/weaveworks/ignite/tree/master/pkg/apis/ignite/v1alpha3/register.go)
+[types.go](https://github.com/weaveworks/ignite/tree/master/pkg/apis/ignite/v1alpha3/types.go)

```

Comment:
These links don't work at the moment, causing the link-checker to fail. Once this PR is merged, these links will become valid and the check should pass.